tests/test_tensor_parallel.py

```diff
@@ -0,0 +1,59 @@
+import random
+
+import pytest
+import torch
+from tensor_parallel import Config, TensorParallel
+from test_utils import MODEL_NAME
+
+from petals.bloom.from_pretrained import load_pretrained_block
+
+
+@pytest.mark.forked
+@pytest.mark.parametrize("custom_config", [True, False])
+@pytest.mark.parametrize("devices", [("cpu",) * 2, ("cpu",) * 3, ("cpu",) * 4])
+def test_tp_block(devices, custom_config):
+    block_index = random.randint(0, 10)
+    block = load_pretrained_block(MODEL_NAME, block_index=block_index, torch_dtype=torch.float32).to(devices[0])
+
+    tp_config = None
+    if custom_config:
+        tp_config = Config(
```

Comment:
Seems like this is the same config as in `convert_block.py`. Please make it a constant in `convert_block.py`, then import it here.

Reply:
moved to core module (tensor_parallel)

Reply:
Can we import it here then?